src/tilt.js

```diff
@@ -37,7 +37,13 @@ class Tilt extends Component {
     this.onMouseLeave = this.onMouseLeave.bind(this, this.props.onMouseLeave);
   }
   componentDidMount() {
-    this.element = findDOMNode(this)
+    this.element = findDOMNode(this);
+    var myNode = this.getDOMNode();
+    setTimeout(() => {
```

Comment:
Why is the `setTimeout` function necessary here?

Reply:
The browser won't have a chance to set :hover state if It's not being called from a setTimeout function. Why should we use setInterval? We just need to run it once